library/proc_macro/src/lib.rs

```diff
@@ -779,16 +779,19 @@ impl !Send for Punct {}
 #[stable(feature = "proc_macro_lib2", since = "1.29.0")]
 impl !Sync for Punct {}
 
-/// Whether an `Punct` is followed immediately by another `Punct` or
-/// followed by another token or whitespace.
+/// Describes whether a `Punct` is followed immediately by another `Punct` ([`Joint`]) or
```

Comment:
```suggestion
/// Describes whether a `Punct` is followed immediately by another `Punct` ([`Spacing::Joint`]) or
```
(And `Spacing::Alone` below.)

Otherwise rustdoc won't be able to resolve links (as CI shows).